src/ga4gh/vrs/extras/translator.py

```diff
@@ -64,39 +56,49 @@ def __init__(
         self.hgvs_tools = None
         self.from_translators = {}
         self.to_translators = {}
+    
+    # def _is_intronic(self, sv: hgvs.sequencevariant.SequenceVariant):
+    #     """Check if HGVS sequence variant is intronic"""
+    #     if isinstance(sv.posedit.pos, hgvs.location.BaseOffsetInterval):
+    #         if sv.posedit.pos.start.is_intronic or sv.posedit.pos.end.is_intronic:
+    #             return True
+    #     return False
+
+    # def _get_hgvs_refget_ac(self, sv: hgvs.sequencevariant.SequenceVariant):
+    #     """Get refget accession for hgvs sequence variant"""
+    #     # prefix accession with namespace
+    #     sequence = coerce_namespace(sv.ac)
+    #     return self._get_refget_accession(sequence)
 
-    @lazy_property
-    def _hgvs_parser(self):
-        """instantiates and returns an hgvs parser instance"""
-        _logger.info("Creating parser")
-        return hgvs.parser.Parser()
+    @staticmethod
+    def _ir_stype(a):
+        """
+        The purpose of this function is to provide a convenient way to extract the sequence type from an accession by matching its prefix to a known set of prefixes.
 
-    def _get_parsed_hgvs(self, hgvs_expr: str):
-        """Get parsed HGVS expression"""
-        if not self.hgvs_re.match(hgvs_expr):
-            return None
+        Args:
+        a (str): The accession string.
 
-        return self._hgvs_parser.parse_hgvs_variant(hgvs_expr)
+        Returns:
+        str or None: The sequence type associated with the accession string, or None if no matching prefix is found.
+        """
 
-    def _get_hgvs_refget_ac(self, sv: hgvs.sequencevariant.SequenceVariant):
-        """Get refget accession for hgvs sequence variant"""
-        # prefix accession with namespace
-        sequence = coerce_namespace(sv.ac)
-        return self._get_refget_accession(sequence)
+        prefix_dict = {
+            "refseq:NM_": "n",
```

Comment:
I think this should be "c"
```suggestion
            "refseq:NM_": "c",
```

Reply:
on it. I ended up moving some of this code to the seqrepo dataproxy classes.